### ISSUE TITLE
Create Rd files for documentation

### ISSUE THREAD

**Issue by olivertam** [olivertam/NDCN_hackathon#1]
Create Rd files with the goal of making all relevant scripts into an R library

**Comment by jules32** [comment]
We can do this with [Roxygen](http://r-pkgs.had.co.nz/man.html), this could be a good Office Hours session for August if folks are interested :)

**Comment by olivertam** [comment]
You're right. I am considering using Roxygen2, though I think it is most useful when we actually have a library, since the Rd files need to be "installed" in the correct location before it is called properly.
Maybe learning how to "package" a bunch of R scripts/functions into a library could be an overarching topic (relevant for open science/sharing/onboarding).
Thanks.

**Comment by olivertam** [comment]
A basic shell of a potential R package has been created for hackathon 3. This might be a good time to revisit this.
